### PR TITLE
(SIMP-3463) Enable metamodule Forge Builds

### PR DESCRIPTION
**Pull request by trevor-vaughan**
This updates simp-core to provide a module that is ready for the Puppet
Forge which should install all other required modules for a base SIMP
build.

SIMP-3463 #close

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -12,6 +12,9 @@ rescue LoadError => e
   warn "WARNING: #{e.message}"
 end
 
+task :metadata_lint do
```
Comment (jeefberkey) [inline Rakefile]: Isn't this in rake helpers?

Comment (trevor-vaughan) [inline Rakefile]: @jeefberkey This is where all of this type of information is housed and this information is identical to the meta-RPM so they should be tightly coupled.

The metadata_lint is in rake helpers but including what is above it enables FAR more rake tasks than we want up here.

File: README.md [hunk 2]
```diff
@@ -1,53 +1,39 @@
...
-## Supported releases
-This branch supports:
-  - RHEL 7.2
-  - [CentOS 7 1511](http://isoredirect.centos.org/centos/7.2.1511/isos/x86_64/CentOS-7-x86_64-DVD-1511.iso)
+## Supported Releases
 
+  - [Red Hat Enterprise Linux 7.3](https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/7.3_Release_Notes/index.html)
+  - [CentOS 7 1611](http://isoredirect.centos.org/centos/7/isos/x86_64/CentOS-7-x86_64-DVD-1611.iso)
+  * [Red Hat Enterprise Linux 6.8](https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/6/html/6.8_Release_Notes/index.html)
+  * [CentOS 6.8](http://mirror.centos.org/centos/6/isos/x86_64/)
```
Comment (jeefberkey) [inline README.md]: 6.0.0-0 released with 6.9

Comment (trevor-vaughan) [inline README.md]: Lovely :-|

**Pull request merged**